protocol/v1/qbft/controller/metrics.go

```diff
@@ -21,17 +27,18 @@ var (
 		Name: "ssv:validator:running_ibfts_count",
 		Help: "Count running IBFTs by validator pub key",
 	}, []string{"pubKey"})
+	metricsDurationAttestation = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "ssv:validator:duration_attestation",
+		Help:    "Attestation duration (seconds)",
+		Buckets: []float64{0.01, 0.05, 0.1, 0.2, 0.5, 1, 2, 5, 10, 60},
```

Comment:
you think we need those high buckets? like the 10 and 60

Reply:
I thought it might be useful to check when consensus is stuck. Until 5s or 10s could mean just 1 missed round. But I don't mind removing 10 and 60

Reply:
above 1.5s it's already not good so i think the heights bucket should be 1.5